core/util.js

```diff
@@ -241,6 +241,47 @@ Util.stopEvent = function (e) {
     e.preventDefault();
 };
 
+// Immediate polyfill
+Util.setImmediate = function (func) {
```

Comment:
I agree that something on a standards track would be more interesting. Unfortunately no such thing exists. :/

We were having similar issues in the generic code at one point, so it ended up in Util. Fortunately we solved those issues another way, so we could move this function if you insist.